app/_locales/es_419/messages.json

```diff
@@ -6,7 +6,7 @@
     "message": "Versión, centro de soporte técnico e información de contacto"
   },
   "acceleratingATransaction": {
-    "message": "* Usar un precio de gas más alto para acelerar una transacción aumenta las posibilidades de un procesamiento más rápido en la red, pero esto no siempre se garantiza."
+    "message": "* Usar un precio de gas más alto para acelerar una transacción aumenta las posibilidades de un procesamiento más rápido en la red, pero esto no siempre se garantiza."
```

Comment:
🤔

Reply:
Looks like this is just correcting a character issue that was presented in the last round of Lionbridge updates: https://github.com/MetaMask/metamask-extension/commit/64e6935558f1623d1a05d9a45ec2595740573fbf#diff-6adb215a14d2f9eb31432edff99fc743d268841e0db7fd9682305dd61f6bfd97R9